pkg/pop/poptokenutils.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package pop
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/confidential"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/public"
+)
+
+func AcquirePoPTokenInteractive(
```

Comment:
Add description on top of each function to describe what it is

Reply:
Added descriptions for all new funcs